dem_sim/model.py

```diff
@@ -10,6 +10,7 @@
 class GNNModel(nn.Module):
     def __init__(
         self,
+        device,
```

Comment:
No, is not possible because in the forward function tensors are created: this requires that we explicitly pass the device otherwise they will be created in default (cpu). 
Lines 90, 91, 92 in model.py

Reply:
For the moment we leave like this, but If we manage to pass the scaling factors as vectors that are in device from train_one_step_wandb.py, then we can remove the parameter in the constructor.